dashboard/pages/api/auth/google/callback.ts

```diff
@@ -0,0 +1,51 @@
+import {NextApiRequest, NextApiResponse} from 'next'
+import {google} from 'googleapis'
+import axios from 'axios'
+import jwt from 'jsonwebtoken'
+
+import {config} from '@lib/config'
+
+export default async function CallbackHandler(
+  req: NextApiRequest,
+  res: NextApiResponse,
+) {
+  const oauth2Client = new google.auth.OAuth2(
+    config.google.clientId,
+    config.google.clientSecret,
+    'http://localhost:3000/api/auth/google/callback',
```

Comment:
Config.hostname instead of localhost 3000

Reply:
```suggestion
    `${config.hostname}/api/auth/google/callback`,
```